scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -47,6 +48,15 @@ object AndroidLintReportFormat extends Enumeration {
 @mill.api.experimental
 trait AndroidAppModule extends JavaModule {
 
+  final def root: Path = super.millSourcePath
+  private def src: Path = root / "src"
+  override def millSourcePath: Path = src / "main"
+
+  override def sources: T[Seq[PathRef]] = Task.Sources(millSourcePath / "java")
+
+  private def bannedModules(classpath: PathRef): Boolean =
+    !classpath.path.last.contains("-jvm")
```

Comment:
This is a bit of a hack so we should leave a comment stating that and linking to the relevant issue https://github.com/com-lihaoyi/mill/issues/3867